content/engineering/languages-runtimes/python.md

```diff
@@ -51,6 +52,10 @@ Our **standard** tool for ensuring consistency across Python code bases is [flak
 
 Use [Black](https://black.readthedocs.io/en/stable/) for automatic code formatting.
 
+We **suggest** using [Ruff](https://github.com/astral-sh/ruff) instead as it
+combines all of the above packages into one and runs faster, however this has
+not yet been adopted widely across TTS yet.
```

Comment:
Suggestion: Reword to be something like "You are welcome to use Ruff instead as it combines all of the above packages into one and runs faster. It maybe become our standard tool in the future."

I worry that saying "not widely adopted" risks making new folks nervous about using it.

Reply:
great idea, thanks!